renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -2562,13 +2567,11 @@ void WrappedOpenGL::CreateVRAPITextureSwapChain(GLuint tex, GLenum textureType,
 
   if(textureType == eGL_TEXTURE_2D_ARRAY)
   {
-    Common_glTextureImage3DEXT(id, eGL_TEXTURE_2D, 0, internalformat, width, height, 2, 0, eGL_RGBA,
-                               eGL_UNSIGNED_BYTE, NULL);
+    Common_glTextureStorage3DEXT(id, eGL_TEXTURE_2D_ARRAY, levels, internalformat, width, height, 2);
```

Comment:
Yeah, now you mention it I think I should change it back. The change was just less typing and less parameters, but I didn't think about the extension might not actually be supported.